ote/src/cljs/ote/views/route/service_calendar.cljs

```diff
@@ -65,7 +65,9 @@
        "Ajopäiväkalenteri"]
```

Comment:
Do we still have some untranslated texts?

Reply:
I'll add it to translations